mediator/esmFldsExchange.F90

```diff
@@ -2118,7 +2118,7 @@ subroutine esmFldsExchange(gcomp, phase, rc)
     !-----------------------------------------------------------------------------
     ! TODO: add this
     ! if (carma_flds /= ' ') then
-    !    do n = 1,shr_string_listGetNum(carma_flds)
+    !    do n = 1,)number_of_fields in carm_flds)
```

Comment:
In comments but wrong as well - remove?

Reply:
I can remove this - but its a place holder to reference what is needed. I can add a comment as well as opposed to pseudo-code. Would that be better?